ephios/settings.py

```diff
@@ -140,9 +147,15 @@
 # Static files (CSS, JavaScript, Images)
 # https://docs.djangoproject.com/en/3.0/howto/static-files/
 
-STATIC_URL = env.str("STATIC_URL")
-STATIC_ROOT = env.str("STATIC_ROOT")
+STATIC_URL = env.str("STATIC_URL", default="/static/")
+STATIC_ROOT = env.str("STATIC_ROOT", default=os.path.join(DATA_DIR, "static"))
```

Comment:
I dont't think that defaulting to data/static is what we want here. The path to the static files should be actively set when installing IMO